### DISCUSSION TITLE
Predict and Analyse YOLOv8 model on xView dataset.

### DISCUSSION THREAD

**Discussion by bshakya77** [obss/sahi#1243] [Q&A] [answered]
Hello, 
I am trying to predict and analyze the YOLOv8n model on xview dataset. I am using the baseline YOLOv8n model pretrained on COCO dataset.  I used the following command:
`!sahi predict \
  --source ./datasets/xView/sliced/subsets/subsets_val_640_50 \
  --dataset_json_path ./datasets/xView/sliced/subsets/annotations/subsets_val_640_50.json \
  --model_type yolov8 \
  --model_path ./models/baseline/yolov8n.pt \
  --slice_height 256 \
  --slice_width 256 \
  --overlap_height_ratio 0.5 \
  --overlap_width_ratio 0.5`

It does not predict anything and mAP is all 0. 

<img width="1266" height="492" alt="image" src="https://github.com/user-attachments/assets/b008f448-6282-4201-9018-bd8c7814687d" />

However when I used the YOLOv8n model trained on xview dataset. It predicts the objects and mAP is as follows on subset of 50 images:

<img width="1334" height="480" alt="image" src="https://github.com/user-attachments/assets/0b9525d8-77d0-4e79-9bff-0e59404fe3cf" />

My question: is it necessary to train the model on the target dataset before prediction and evaluation or we can use the COCO trained model using transfer learning.

Thank you.

Regards,
Bijay

**Comment by fcakyon** [comment, marked as answer]
@bshakya77 COCO and xView dataset labels and context are very different (xView images are taken in a birdseye-view from satellite, while COCO images are taken from ground closer to target objects). 

Thus, in this case, you need to finetune the baseline model on xView.

**Reply by bshakya77** [reply]
Okay, I got it. 
Thank you for your information.
